appengine/Dockerfile

```diff
@@ -55,9 +55,10 @@ ENV PATH /rbenv/shims:/rbenv/bin:$PATH
 
 # Preinstall ruby runtimes.
 # The LAST version in the list is set as the default.
-ENV PREINSTALLED_RUBY_VERSIONS 2.0.0-p648 2.1.8 2.2.3 2.2.4 2.3.0
+ENV PREINSTALLED_RUBY_VERSIONS 2.1.8 2.2.4 2.3.0
```

Comment:
nit - in the future, can you separate these changes into different one-purpose commits?